native_client/BUILD

```diff
@@ -51,12 +51,28 @@ tf_cc_shared_object(
     copts = ["-Wno-sign-compare", "-fvisibility=hidden"],
     linkopts = select({
         "//tensorflow:darwin": [],
-        "//conditions:default": [
-            "-ldl",
-            "-pthread",
-            "-Wl,-Bsymbolic",
-            "-Wl,-Bsymbolic-functions",
-            "-Wl,-export-dynamic",
+        "//tensorflow:linux_x86_64": [
```

Comment:
nit: maybe we should factorize the common flags into a variable ?

Reply:
Will do.